cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,92 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import faker from 'faker';
+import {
+  updateSuccessMessage,
+  confirmationMessage
+} from '../plugins/alertMessages';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('db:clear');
   });
```

Comment:
Database clearing should be put in beforeEach hook to ensure clean state for each test.

Reply:
Done